tracer/test/snapshots/DataStreamsMonitoringAwsSqsTests.SubmitsDsmMetrics.single.sameThread.injection.verified.txt

```diff
@@ -0,0 +1,70 @@
+﻿{
+  Env: integration_tests,
+  Service: Samples.AWS.SQS,
+  TracerVersion: <snip>,
+  Lang: dotnet,
+  Stats: [
+    {
+      Start: 1661520120000000000,
+      Duration: 10000000000,
+      Stats: [
+        {
+          EdgeTags: [
+            direction:out,
+            topic:MyAsyncSQSQueue,
+            type:sqs
+          ],
```

Comment:
I wonder if we need to sort these Stats fields by `Hash` I see that in some cases the `in` edges are first and in others the `out` edges are first 🤔 Seems like it could be a source of some flake?

Reply:
I don't think it can flake because of this since the hashes are stable (and there is a defined sorting order).
The rule is that the parents go first, I think the `in` can appear at the top for the cases where injection is blocked (by having already all 10 headers occupied).